backend/api/oauthApi.test.js

```diff
@@ -95,21 +95,34 @@ describe('oauthApi tests', () => {
 
   describe('getUserEmail', () => {
     const emailDetails = { email: 'hello@there', username: 'someuser' }
-    let actual
 
     beforeEach(() => {
       client.get = jest.fn().mockReturnValue({
         then: () => emailDetails,
       })
-      actual = oauthApi.getUserEmail(context, { username: 'joe' })
     })
 
-    it('should return email from endpoint', () => {
+    it('should return email from endpoint', async () => {
+      const actual = await oauthApi.getUserEmail(context, { username: 'joe' })
       expect(actual).toEqual(emailDetails)
     })
     it('should call user email endpoint', () => {
+      oauthApi.getUserEmail(context, { username: 'joe' })
       expect(client.get).toBeCalledWith(context, '/api/user/joe/email')
     })
+    it('should cope with not found from endpoint', async () => {
+      const error = new Error('User not found')
+      // @ts-ignore
```

Comment:
If you wanted to get rid of this ignore you may be able to do: `const error = {...new Error('User not found'), status: 404)`

Reply:
cunning :grin: